OakCityV2/DetailViewController.m

```diff
@@ -12,60 +12,24 @@
 @implementation DetailViewController
 
 
--(void)viewDidLoad
-{
-    [_titleLabel loadHTMLString:self.ttitle baseURL:nil];
-    [_companyLabel loadHTMLString:self.company baseURL:nil];
-    [_urlLabel loadHTMLString:self.url baseURL:nil];
-    _urlLabel.dataDetectorTypes = UIDataDetectorTypeLink;
-    [_descriptionLabel loadHTMLString:self.desc baseURL:nil];
+-(void)viewDidLoad {
```

Comment:
There's a bug in here that's not obvious.  You should run the Static Analyzer and see what it says.  

Xcode Menu -> Product -> Analyze

Here's how to run the analyzer with every build.
http://matthewmorey.com/automatic-static-analysis/